PWGCF/Flow/Tasks/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ o2physics_add_dpl_workflow(flow-pt-efficiency
                     COMPONENT_NAME Analysis)
 
 o2physics_add_dpl_workflow(flow-pbpb-task
-                    SOURCES FlowPbPbTask.cxx
+                    SOURCES FlowTask.cxx
```

Comment:
Could you please also modify accordingly the name of the executable, `flow-pbpb-task` to `flow-task`

Reply:
sure